src/quo2/components/channel_avatar.cljs

```diff
@@ -0,0 +1,43 @@
+(ns quo2.components.channel-avatar
+  (:require [quo2.foundations.colors :as colors]
+            [quo.react-native :as rn]
+            [quo2.components.icon :as icons]
+            [quo2.components.text :as text]))
+
+(defn channel-avatar [{:keys [big? dark? lock-status avatar-color avatar-text]}]
```

Comment:
`avatar-text` will always be emoji, should we rename it just `emoji` ?

Reply:
ok